Lib/test/test_tools/test_makefile.py

```diff
@@ -0,0 +1,66 @@
+"""
+Tests for `Makefile`.
+"""
+
+import os
+import unittest
+from test import support
+
+MAKEFILE = os.path.join(support.REPO_ROOT, 'Makefile')
```

Comment:
This doesn't find the `Makefile` in out-of-tree builds. IMO,y this should be `sysconfig.get_makefile_filename()`.

```suggestion
import sysconfig

MAKEFILE = sysconfig.get_makefile_filename()
```